hostinfo/hostinfo_windows.go

```diff
@@ -8,11 +8,12 @@ import (
 	"fmt"
 	"os"
 	"path/filepath"
-	"strings"
 	"sync/atomic"
 
 	"golang.org/x/sys/windows"
 	"golang.org/x/sys/windows/registry"
+
```

Comment:
you can delete this blank line. we generally only do two sections: std vs not.

We'd only do one section but then it looks weird with "github.com/long/long/long/long" sorting between short "fmt" and "io"